### ISSUE TITLE
Add ability for LOST to replay Valhalla routes

### ISSUE THREAD

**Issue by msmollin** [mapzen/android#464]
Maybe have a shim layer to convert routes to GPX traces in the SDK?

**Comment by olivier65** [comment]
Yes. This will waste resources but I understand the desire to keep the LOST and On-the-Road projects separate.

**Comment by msmollin** [comment]
One thought that came to us later was adding support for generic polyline playback (Valhalla routes are just polylines with some additional metadata around them) - so then LOST itself can support mocking polylines in a generic fashion and our Mapzen Android SDK can provide the middleware layer for connecting OnTheRoad into LOST properly.

Thoughts?
